data/view_in_a_map/lab.md

```diff
@@ -21,4 +21,4 @@ In this lab you will add your own layers and some existing publicly available la
 6. Save your map. 
 
 ###Bonus
-* In the Map Viewer select `Add` > `Search for Layers` > `In: ArcGIS Online` to load other pre-existing layers into your map. HINT: Try searching for weather, rain or wind.
\ No newline at end of file
+* In the Map Viewer select `Add` > `Search for Layers` > `In: ArcGIS Online` to load other pre-existing layers into your map. HINT: Try searching for weather, rain or wind. You might try changing `In: My Organization` to `In: ArcGIS Online` to see more layers.
```

Comment:
searching across ArcGIS Online will definitely return more results than a search within an individual Organization.  

i was only pointing out that your current wording seems unclear.  the original step asks them to select `In: ArcGIS Online` and then you mention switching from `In: My Organization` to `In: ArcGIS Online` (where they already should have been)